contrib/debian/rules

```diff
@@ -6,7 +6,7 @@
 #	$(if $(filter nocheck,$(DEB_BUILD_OPTIONS)),,src/test_bitcoin)
 
 DEB_INSTALL_EXAMPLES_bitcoind += debian/examples/*
-DEB_INSTALL_MANPAGES_bitcoind += debian/manpages/*
+DEB_INSTALL_MANPAGES_bitcoind += doc/man/*
```

Comment:
No, mostly this is a flag for a build method that we dont use - so these variables are just entirely ignored :).